diffkemp/simpll/DebugInfo.cpp

```diff
@@ -20,6 +20,8 @@ using namespace llvm;
 
 PreservedAnalyses RemoveDebugInfoPass::run(Function &Fun,
                                            FunctionAnalysisManager & /*fam*/) {
+    DEBUG_WITH_TYPE(DEBUG_SIMPLL,
+                    dbgs() << "Analysing debugging information...\n");
```

Comment:
This should go to `DebugInfo` (probably the best place would be its constructor).

Reply:
Does that mean creating a destructor and putting `decreaseDebugIndentLevel()` in there?

Reply:
Most of the analysis is done in the constructor, so I'd put `decreaseDebugIndentLevel()` to its end.